### PR TITLE
Add filter to debug UI card list.

### PR DESCRIPTION
**Pull request by kberg**
filter

### CODE REVIEW

File: src/components/DebugUI.ts [hunk 1]
```diff
@@ -16,17 +21,20 @@ export const DebugUI = Vue.component("debug-ui", {
...
+        },
+        filtered: function(cardName: string):boolean {
+            return this.$data.filterText.length == 0 || cardName.toUpperCase().indexOf(this.$data.filterText.toUpperCase()) > -1;
         }
     },
     template: `
         <div class="debug-ui-container">
             <section class="debug-ui-cards-list">
-                <h2>Cards list</h2>
+                <h2>Cards list <input placeholder="filter" v-model="filterText"></h2>
```
Comment (alrusdi) [inline src/components/DebugUI.ts:35]: Add classes "form-input form-inline" to the input

File: src/components/DebugUI.ts [hunk 2]
```diff
@@ -7,6 +7,11 @@ export const DebugUI = Vue.component("debug-ui", {
     components: {
         Card,
     },
+    data: function() {
+        return {
+          filterText: ""
```
Comment (alrusdi) [inline src/components/DebugUI.ts:12]: Can you take initial value of filterText from URL like "/debug-ui/?search-card=venusian" ahen component is "mounted"?

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/components/DebugUI.ts (hunk 2 above)
Comment (kberg) [reply src/components/DebugUI.ts:12]: I didn't realize this was holding up the review, sorry. Can that be done in a follow-up? I'm not sure I want to start diving into URL parameters yet.

File: src/components/DebugUI.ts (hunk 1 above)
Comment (alrusdi) [reply src/components/DebugUI.ts]: @kberg and what about this change?

Comment (kberg) [reply src/components/DebugUI.ts]: > Add classes "form-input form-inline" to the input
Done, good, thanks!

> @kberg and what about this change?

My head doesn't have that anymore. Still an issue?

**Review by alrusdi — APPROVED** [review]

**Pull request merged**